.github/actions/setup-spack-stack/action.yaml

```diff
@@ -49,6 +55,15 @@ runs:
             sudo apt-get update
             sudo apt-get install intel-oneapi-dev-utilities intel-oneapi-openmp intel-oneapi-compiler-fortran intel-oneapi-compiler-dpcpp-cpp-and-cpp-classic intel-oneapi-mpi-devel
             echo "source /opt/intel/oneapi/setvars.sh" >> ~/.bash_profile
+            #
+            # DH*
+            # For now, install qt5 only for Intel to shorten the build time
+            # without having to rerun all other CI tests. Later, should consider
+            # using https://github.com/jurplel/install-qt-action for all runners
+            # and get rid of the macOS homebrew installation/package config
+            #sudo apt-get install qt5-default qttools5-dev-tools
```

Comment:
Note. These are not yet used in spack, there was a problem during the concretization step when I added it manually as an external spack. I worked around the long build times using package-level plus install-level parallelism, see further down.